superset/assets/javascripts/explore/components/ExploreViewContainer.jsx

```diff
@@ -101,13 +106,37 @@ class ExploreViewContainer extends React.Component {
     }
   }
 
+  addHistory(isReplace = false) {
+    const { url, payload } = createExplore(this.props.form_data);
+    if (isReplace) {
+      history.replaceState(
```

Comment:
I can't figure out where `history` is defined.

Reply:
`history` is `window.history`, html5 Web API:
https://developer.mozilla.org/en-US/docs/Web/API/Window/history